js/main.js

```diff
@@ -1 +1,94 @@
+const MAX_PHOTOS = 25;
+const MIN_LIKES = 15;
+const MAX_LIKES = 200;
+const MIN_COMMENTS = 0;
+const MAX_COMMENTS = 30;
+const MIN_AVATARS = 1;
+const MAX_AVATARS = 6;
 
+//getting random number from stated range
+const getRandomNumber = (a, b) => {
+	const lower = Math.ceil(Math.min(a, b));
+	const upper = Math.floor(Math.max(a, b));
+	const result = Math.random() * (upper - lower + 1) + lower;
+	return Math.floor(result);
+};
+
+//getting random element from stated array
+const getRandomArrayEllement = (elements) => elements[getRandomNumber(0, elements.length - 1)];
```

Comment:
Опечатка
```suggestion
/** getting random element from stated array */
const getRandomArrayElement = (elements) => elements[getRandomNumber(0, elements.length - 1)];
```
Либо более полные типы
```suggestion
/**
 * getting random element from stated array
 * @template Element
 * @param {Element[] | readonly Element[]} elements
 */
const getRandomArrayElement = (elements) => elements[getRandomNumber(0, elements.length - 1)];
```